internal/handlers/common/aggregations/number.go

```diff
@@ -19,13 +19,13 @@ import (
 	"math/big"
 )
 
-// sumNumbers accumulate numbers and returns the result of summation.
+// SumNumbers accumulate numbers and returns the result of summation.
```

Comment:
As this function has only one usage with the `sum` operator, should we place it somewhere near its implementation?

Reply:
moved to `operators` package.msg_